src/encoded/static/components/genome_browser.js

```diff
@@ -0,0 +1,432 @@
+import React from 'react';
+
+
+const maxFilesBrowsed = 5; // Maximum number of files to browse
+const domainName = 'https://www.encodeproject.org';
+const dummyFiles = [
```

Comment:
I was working on a branch of peak_indexer that had a few local bed files for testing purposes.